modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java

```diff
@@ -454,11 +454,12 @@ void addLiveTracks(MediaPackage mp, String caName) throws LiveScheduleException
       }
 
       // Capture agent did not pass any CA_PROPERTY_RESOLUTION_URL_PREFIX property when registering
-      // so use the service configuration, including stream overrides (temporary)
-      if (liveStreamingUrl == null)
-        throw new LiveScheduleException(
-                "Cannot build live tracks because '" + LIVE_STREAMING_URL + "' configuration was not set.");
+      // so use the service configuration
       if (mp.getTracks().length == 0) {
+        if (liveStreamingUrl == null)
+          throw new LiveScheduleException(
+                  "Cannot build live tracks because '" + LIVE_STREAMING_URL + "' configuration was not set.");
+
```

Comment:
The original ticket was created because of a problem in the live scheduler service. Later, I realized that the problem was caused by the metadata. That's why I thought it was OK to fix the bug in the same PR.